lcls_tools/common/devices/ict.py

```diff
@@ -0,0 +1,35 @@
+from pydantic import field_validator, SerializeAsAny
+
+from lcls_tools.common.devices.device import Device, PVSet, ControlInformation, Metadata
+from epics import PV
+
+
+class ICTPVSet(PVSet):
+    """
```

Comment:
we would need to update the reader script to pull ICT in yaml files too. Do you have a target ICT/name I could add?

Reply:
no idea